WalletWasabi.Fluent/ViewModels/AddWallet/ImportWalletViewModel.cs

```diff
@@ -23,20 +19,20 @@ public class ImportWalletViewModel : RoutableViewModel
 
 		public ImportWalletViewModel(string walletName, WalletManager walletManager)
 		{
+			Title = "Import Wallet";
 			WalletName = walletName;
 			WalletManager = walletManager;
 
-			ImportWallet();
+			Task.Run(ImportWallet);
```

Comment:
This should be done in override OnNavigatedTo ()

Reply:
and awaited, and scheduled on UIThread...

RxApp.MainThreadScheduler.Schedule(async () => ....